docs/data/data-grid/row-grouping/row-grouping.md

```diff
@@ -228,41 +241,50 @@ const columns: GridColDef[] = [
 If your column also has a `valueGetter` property, the value passed to the `groupingValueGetter` method will still be the row value from the `row[field]`.
 :::
 
-## Rows with missing groups
+### Rows with missing groups
 
-If the grouping key of a grouping criteria is `null` or `undefined` for a row, the Data Grid will consider that this row does not have a value for this group. and will inline it for those groups.
+If a grouping criterion's key is `null` or `undefined` for a given row, the Data Grid will treat that row as if it doesn't have a value and exclude it from grouping.
+
+The demo below illustrates this behavior—movies are grouped by **Cinematic Universe**, and rows with no value for this column are displayed individually before those with values are displayed in groups:
 
 {{"demo": "RowGroupingRowsWithMissingGroups.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## Group expansion
+## Expanding groups on first load
```

Comment:
I'd revert this change since it's not necessarily on the first load, it technically happens whenever the rows pre-processor runs which could happen multiple times during the lifecycle of a Data Grid.

Reply:
thanks for clarifying!